inspire/config.py

```diff
@@ -230,6 +230,7 @@
     "reportnumber": ["report_number.value"],
     "experiment": ["accelerator_experiment.experiment"],
     "title": ["title.title"],
+    "department_acronym": ["department_acronym"],
```

Comment:
Hmmmmm you are maybe right but I let me check one second and I will inform
you.

2015-10-06 17:22 GMT+02:00 Javier Martin Montull <notifications@github.com>:

> In inspire/config.py
> <https://github.com/inspirehep/inspire-next/pull/396#discussion_r41278606>
> :
>
> > @@ -230,6 +230,7 @@
> >      "reportnumber": ["report_number.value"],
> >      "experiment": ["accelerator_experiment.experiment"],
> >      "title": ["title.title"],
> > +    "department_acronym": ["department_acronym"],
>
> Did you try if this is needed? Usually you need to configure it in cases
> where the value is in a second level like
> accelerator_experiment.experiment but for first level I thought it worked
> by default.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/396/files#r41278606>.
>